CODEWALK.md

```diff
@@ -636,3 +640,20 @@ Our implementation of [plumbing and porcelain](#plumbing-and-porcelain) embraces
 
 This idiom is unfortunately hidden away in a [wiki page about code review](https://github.com/golang/go/wiki/CodeReviewComments#interfaces).
 See also Dave Cheney on [SOLID Go Design](https://dave.cheney.net/2016/08/20/solid-go-design).
+
+### Observability
+
+go-filecoin uses [Opencensus-go](https://github.com/census-instrumentation/opencensus-go) for stats collection and distributed tracing instrumentation.
+Stats are exported for consumption via [Prometheus](https://prometheus.io/) and traces are exported for consumption via [Jaeger](https://www.jaegertracing.io/docs/1.11/).
+
+#### Metrics
+
+go-filecoin can be configured to collect and export metrics to Prometheus via the `MetricsConfig`.
+The details of this can be found inside the [`config/`](https://godoc.org/github.com/filecoin-project/go-filecoin/config#ObservabilityConfig) package.
+To view metrics from your filecoin node using the default configuration options set the `prometheusEnabled` value to `true`, start the filecoin daemon, then visit `localhost:9400/metrics` in your web-browser. 
```

Comment:
I would prefer to set it to `true` during the deployment. Because we run all integration tests in parallel, setting this to true by default will cause them to fail due to port conflicts, i.e:
`ERROR metrics: failed to serve /metrics endpoint on listen tcp 0.0.0.0:9400: bind: address already in use export.go:58`. Some solutions could be, using port `0` by default, randomly assign the port at testing time, or disabling metrics during testing. I believe defaulting to `false` is the simplest solution here.